apps/cookbook/src/app/shared/_example-frame.scss

```diff
@@ -14,4 +14,8 @@
   &.no-padding {
     padding: 0;
   }
+
+  &.xl-padding {
+    padding: utils.size('xl');
+  }
```

Comment:
Just revisited the comments from @henrikvoetmand, and turns out all exampe frames should have `l` padding, so this class is not needed at all!